test/unit_testing/bloc_utils_test.dart

```diff
@@ -0,0 +1,125 @@
+import '../../lib/src/core/utils/bloc_utils.dart';
+import '../../lib/src/core/utils/state.dart';
+import '../../lib/src/core/utils/ui_resource_state.dart';
+import '../../lib/src/domain/entities/genre.dart';
+import '../../lib/src/domain/entities/movie.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+List<Movie> movies = [];
+DataState successMovie = DataState(
+  responseState: ResponseState.success,
+  data: movies,
+);
+List<Genre> genres = [];
+DataState successGenre = DataState(
+  responseState: ResponseState.success,
+  data: genres,
+);
+DataState error = DataState(responseState: ResponseState.error);
+DataState empty = DataState(responseState: ResponseState.empty);
+DataState loading = DataState(responseState: ResponseState.loading);
```

Comment:
Move these inside the main method, or make them file-private (prepending an '_')

Reply:
Ok! Done!